src/components/Employees/Employees.module.css

```diff
@@ -0,0 +1,89 @@
+@import url('../../theme.module.css');
+
+.employees {
+    display: flex;
+    flex-direction: column;
+    background-color: var(--page-background-color);
+    color: white;
+    gap: 30px;
+    padding: 40px;
+    height: 100vh;
+    width: 87vw;
```

Comment:
87 seems a bit random, also can you please use % instead of vh and vw as they can introduce bugs

Reply:
I have problem when i get less then 4 results while typing in search input, it changes my width and makes it ugly. I cannot figure it out what is causing it.